demisto_sdk/commands/download/downloader.py

```diff
@@ -1274,113 +1301,45 @@ def update_data(output_path: str, file_path_to_read: str, file_ending: str) -> N
             splitter="dot",
         )
 
-        file_data = get_file(output_path, clear_cache=True)
+        file_data = get_file(file_to_update)
+
         if pack_obj_data:
-            merge(file_data, preserved_data)
+            mergedeep.merge(file_data, preserved_data)
 
-        if file_ending == "yml":
-            safe_write_unicode(lambda f: yaml.dump(file_data, f), Path(output_path))
-        elif file_ending == "json":
+        if is_yaml:
+            safe_write_unicode(lambda f: yaml.dump(file_data, f), file_to_update)
+        else:  # json
             safe_write_unicode(
-                lambda f: json.dump(data=file_data, fp=f, indent=4), Path(output_path)
+                lambda f: json.dump(data=file_data, fp=f, indent=4), file_to_update
             )
-        else:
-            raise RuntimeError(f"cannot merge file extension {file_ending}")
```

Comment:
bring this back

Reply:
There's no point, there is no more `file_ending` variable here, and files passed to this function are either YAML or JSON (indicated by the `is_yaml` parameter).